docs/ahds.rst

```diff
@@ -1,15 +1,32 @@
 ``ahds`` package
 ================
 
-.. automodule:: ahds
+.. automodule:: src.ahds
```

Comment:
I suppose this is for practical purposes i.e. it only works when pointing to `src`

Reply:
Ok i have checked this was necessary cause in the pull request #4 which you already have merged by the commit [964eff1](https://github.com/emdb-empiar/ahds/pull/4/commits/964eff1bc3b94877217de5688c6300d8ede747d8) i had to change file system structure to consistently be able to reproduce a bug locally using pytest and tox which i encountered on the CI (back then still Travis). The only way back then i found to get hold of the bug without overstressing CI was to change filesystem structure to the one preferred by pytest. This now reflects this previous change upon doc sources.